README.md

```diff
@@ -22,8 +22,8 @@ For more information on the reliable web app pattern, see [Overview](https://rev
 
 [![Diagram showing the architecture of the reference implementation](docs/assets/reliable-web-app-java.svg)](docs/assets/reliable-web-app-java.svg#lightbox)
 
-- [Production environment estimated cost](https://azure.com/e/a44f5feb443f430abbd9116b6cc879bf)
-- [Non-production environment estimated cost](https://azure.com/e/807cdb0d6d9a41899885bd875deb12f5)
+- [Production environment estimated cost](https://azure.com/e/9ed08d9eb2ba426e95e4e7f5a8b075e7)
+- [Non-production environment estimated cost](https://azure.com/e/b6ac4fdb62f3475cb1c2dee388dc4b9a)
```

Comment:
```suggestion
- [Non-production environment estimated cost](https://azure.com/e/1721b2f3f2bd4340a00115e79057177a)
```
Apply Dev/Test pricing for non-prod and use smaller (50gB) storage estimate for non-prod.